src/Domain/Package.php

```diff
@@ -84,6 +97,6 @@ public static function is_experimental_build() {
 	 * @return boolean
 	 */
 	public static function is_feature_plugin_build() {
-		return WOOCOMMERCE_BLOCKS_PHASE > 1;
+		return self::feature()->is_feature_plugin_build();
 	}
```

Comment:
`is_experimental_build` and `is_feature_plugin_build` should not be static methods (I realize this wasn't introduced in this pr).